.bit/responses/5.1-Week 5 Step 1.md

```diff
@@ -0,0 +1,46 @@
+---
+files: blog.md
+stepType: PRmerge
+scripts: n/a
+week: 5
+step: 1
+name: Determine the Scope of your Project
+---
+
+## Determine the Scope of your Project
+
+Now that we have successfully completed learning about Serverless architecture and practiced using Azure functions, let's get creative and start building out our project.
+
+The ultimate goal of this project is to develop a solution for a real world problem within one of the three following categories:
+
+🌍 Earth, environment, and sustainability
+
+💚 Productivity, health, and medicine
+
+🎨 Entertainment, art, and community
+
+---
+
+Here are some leading questions you can ask yourself to help make a decision:
+
+🧩 What types of games/shows/movies/genres do you find interesting?
+
+🎒 What are some struggles that you have faced in life/in the last year?
+
+⚡️ What are some struggles your friends/family have faced?
+
+💻 What types of apps do you like interacting with?
+
+🗣 What type of impact do you want to leave in the world?
+
+🤝 When in doubt, ask your friends/mentors/teachers/professors/co-workers for advice!
+
+[Here are some example blogs written by students in the past!](https://bitproject.org/blog)
+
+---
+
+## :pencil: Task: What problem does your project tackle? How does this correlate to the Conference themes?
+
+- [ ] Create a new branch called `week5`
```

Comment:
these checkboxes are amazing - should we implement these for week the other steps?? @lazyplatypus